src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c

```diff
@@ -116,7 +200,7 @@ void EnKendoJs_Init(Actor* thisx, PlayState* play) {
     Collider_SetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
     CollisionCheck_SetInfo2(&this->actor.colChkInfo, DamageTable_Get(0x16), &sColChkInfoInit);
 
-    SkelAnime_InitFlex(play, &this->skelAnime, &object_js_Skel_006990, &object_js_Anim_000F4C, this->jointTable,
+    SkelAnime_InitFlex(play, &this->skelAnime, &object_js_Skel_006990, &gSwordmasterDefaultAnim, this->jointTable,
                        this->morphTable, OBJECT_JS_LIMB_MAX);
 
     if ((CURRENT_DAY == 3) && ((CURRENT_TIME > CLOCK_TIME(23, 0)) || (CURRENT_TIME < CLOCK_TIME(6, 0)))) {
```

Comment:
Do you think you understand this actor enough to rename `ENKENDOJS_GET_FF` and `ENKENDOJS_FF_1`?

Reply:
`_FF00` is already done. It's a "path index" which is kind of a misnomer; in reality, it indexes the setup path list for that actor, which determines the spawn locations of logs under the floor.

There are two `KendoJs` actors that spawn: one in the main room and one in the back room. `_FF` determines which is which, so I've renamed it to "location".

Reply:
Generally different behaviors are called different "Types" but location seems fine too.